metricbeat/module/prometheus/remote_write/_meta/docs.asciidoc

```diff
@@ -7,6 +7,10 @@ remote_write:
   - url: "http://localhost:9201/write"
 ------------------------------------------------------------------------------
 
+TIP: It is recommended to carefully choose the https://prometheus.io/docs/practices/remote_write/#parameters[parameters]
+of remote_write configuration so as to tune the queue according to each specific case and system's characteristics.
```

Comment:
I think this should be more specific, focusing on the ones that affect behavior and explaining them. Specially the relation between system fd limits (`ulimit`) and the max shards setting

Reply:
This would be helpful, indeed @exekias. Thanks!